mk/config.mk

```diff
@@ -205,6 +205,21 @@ CFG_WITH_USER_TA ?= y
 # By default, in-tree TAs are built using the first architecture specified in
 # $(ta-targets).
 
+# Address Space Layout Randomization for user-mode Trusted Applications
+#
+# When this flag is enabled, the ELF loader will introduce a random offset
```

Comment:
maybe s/`random offset`/`random page offset`/

Reply:
Ok. Actually `CFG_TA_ASLR_*_OFFSET_PAGES` inline comment below give the required details about offset value. So fine to me as is.